leaflet/index.d.ts

```diff
@@ -1524,6 +1524,26 @@ declare namespace L {
         export const svg: boolean;
     }
 
+    export namespace Util {
+        export function extend(dest: {}, src?: {}): {};
```

Comment:
I used `{}` based on the Typescript linter.

```
1539:45  forbidden-types  Avoid using the Object type. Did you mean `any` or `{}`?
```
However I do agree it looks kinda weird :) I'll use `any` instead